src/playbin.js

```diff
@@ -44,41 +58,33 @@ var Playbin = GObject.registerClass({
         );
         this._videocrop = this._playbin.videoFilter.get_by_name('videocrop');
 
-        this._playbin.videoSink.showPrerollFrame = false;
         this._video_widget = this._playbin.videoSink.widget;
         this._video_widget.expand = true;
         this._video_widget.noShowAll = true;
         this._video_widget.ignoreAlpha = false;
-        this._video_widget.hide();
+        this._video_widget.show();
         this.add(this._video_widget);
 
-        this._video_widget.connect('size-allocate', this._updateCropArea.bind(this));
-
         this._playbin.get_bus().add_watch(0, this._bus_watch.bind(this));
     }
 
-    _updateCropArea () {
-        const width = this._video_widget.get_allocated_width();
-        const height = this._video_widget.get_allocated_height();
-
-        if (width < 2 || height < 2 || !this._video_width || !this._video_height)
+    _updateCropArea (alloc) {
+        if (!this._videocrop ||
+            !this._video_width || !this._video_height ||
+            alloc.width < 2 || alloc.height < 2)
             return;
 
-        const crop_x = (this._video_width - width) / 2;
-        const crop_y = (this._video_height - height) / 2;
+        const crop_x = (this._video_width - alloc.width) / 2;
+        const crop_y = (this._video_height - alloc.height) / 2;
 
         this._videocrop.left = crop_x;
         this._videocrop.right = crop_x;
         this._videocrop.top = crop_y;
         this._videocrop.bottom = crop_y;
-
-        if (!this._started && this._state === Gst.State.PLAYING) {
-            this._started = true;
-            this.emit('start');
-        }
     }
 
-    _updateSizeFromStreamCollection (collection) {
+    _onStreamsSelected (msg) {
+        var collection = msg.parse_streams_selected();
```

Comment:
Should this not be `const`?

Reply:
I wonder why the linter in travis CI didn't catch this...